src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { useTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import MobileStepper from "@mui/material/MobileStepper";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import KeyboardArrowLeft from "@mui/icons-material/KeyboardArrowLeft";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import SwipeableViews from "react-swipeable-views";
+import { autoPlay } from "react-swipeable-views-utils";
+import { SLIDER_IMAGES } from "../../common/constants";
+
+const AutoPlaySwipeableViews = autoPlay(SwipeableViews);
+
+function Slider() {
```

Comment:
you can just type here `export default function Slider(){...}` and then you can remove `export default `at the bottom
And add please TS for functional component. It can be for example Slider(): React.ReactElement {...}